.github/workflows/ci.yml

```diff
@@ -32,8 +32,7 @@ jobs:
     continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
-        rust: [stable, beta]
```

Comment:
Does it make sense to remove beta but keep nightly ?

Reply:
Or maybe I'm misunderstanding what L40 does actually

Reply:
Yeah, I removed it